.github/workflows/compound-java-dev.yml

```diff
@@ -52,6 +52,21 @@ on:
         type: boolean
         default: false
         description: 'false for dependencyCheckAnalyze, true for dependencyCheckAggregate'
+      integration-test-enabled:
+        required: false
+        type: boolean
+        default: true
```

Comment:
issue: Why the default value is `true`? Not all our projects have integration tests. If there is no test task to execute the workflow will fail